c2corg_ui/templates/helpers/view.html

```diff
@@ -239,8 +229,23 @@ <h3 class="heading show-phone" ng-click="mainCtrl.animateHeaderIcon($event)" dat
         <div class="list-item" ng-repeat="doc in ${associations}" ng-class="{'new-item': doc['new']}">
           <app-card app-card-doc="doc"></app-card>
           % if type == 'books':
-            <app-delete-association child-doctype="books" parent-id="${document['document_id']}" child-id="doc.document_id"
-              ng-if="userCtrl.auth.isAuthenticated()"></app-delete-association>
+              <app-delete-association child-doctype="books" parent-id="${document['document_id']}" child-id="doc.document_id"
+                ng-if="userCtrl.auth.isAuthenticated()"></app-delete-association>
+          % endif
+          % if is_collaborative(document):
+              && userCtrl.auth.isModerator()
```

Comment:
I don't understand this code: to what will ``&& userCtrl.auth.isModerator()`` be added?